pom.xml

```diff
@@ -31,8 +31,47 @@
         <!-- Generate PackageVersion.java into this directory. -->
         <packageVersion.dir>org/openapitools/jackson/nullable</packageVersion.dir>
         <packageVersion.package>${project.groupId}.jackson.nullable</packageVersion.package>
+        <java.version>1.8</java.version>
+
+        <!-- region Dependency Versions -->
+        <!-- Validator 8+ requires Java 11, Validator 9+ requires Java 17. -->
+        <hibernate-validator.version>7.0.5.Final</hibernate-validator.version>
+        <jackson-bom.version>2.20.0</jackson-bom.version>
+        <javax-validation-api.version>2.0.1.Final</javax-validation-api.version>
+        <jakarta-validation-api.version>3.1.1</jakarta-validation-api.version>
+        <junit.version>5.13.4</junit.version>
+        <!-- endregion -->
+
+        <!-- region Plugin Versions -->
+        <central-publishing-maven-plugin.version>0.9.0</central-publishing-maven-plugin.version>
+        <maven-compiler-plugin.version>3.14.1</maven-compiler-plugin.version>
+        <maven-gpg-plugin.version>3.2.8</maven-gpg-plugin.version>
+        <maven-jar-plugin.version>3.4.2</maven-jar-plugin.version>
+        <maven-javadoc-plugin.version>3.12.0</maven-javadoc-plugin.version>
+        <maven-replacer-plugin.version>1.5.3</maven-replacer-plugin.version>
+        <maven-source-plugin.version>3.3.1</maven-source-plugin.version>
+        <!-- endregion -->
     </properties>
 
+    <dependencyManagement>
```

Comment:
This section is to aid in the migration away from `jackson-base` longer term. Instead of relying on the versions provided by the parent, provide the versions ourselves here.